src/styles/styles.js

```diff
@@ -1531,6 +1531,10 @@ const styles = {
         cursor: 'not-allowed',
     },
 
+    rejectIOU: {
+        minWidth: '20%',
+    },
```

Comment:
We shouldn't create unique style objects for a single style like this. 

Instead, I think you should create a new sizing utility, like this example: https://github.com/Expensify/Expensify.cash/blob/3ebf6136517de6f9ad101037392e2a0a5954347c/src/styles/utilities/sizing.js#L19-L21